js/app.js

```diff
@@ -64,29 +62,22 @@ function createRandomImages(){
         }
```

Comment:
Since you have the `Product` objects stored in the `displayArray`, you can just loop through the `displayArray` and set `displayArray[i].previous = false;`.